src/EscapeHit.Invoice.Database/InvoiceDbContext.cs

```diff
@@ -20,6 +25,15 @@ public override ValueTask DisposeAsync()
             return base.DisposeAsync();
         }
 
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            base.OnModelCreating(modelBuilder);
+
+            var invoiceEntitySpecification = _serviceLocator.GetService<InvoiceEntitySpecification>();
```

Comment:
At the moment it is a manual activity to query entity specifications. We need to evolve it the foreseeable future and automate this process.